### DISCUSSION TITLE
Index page not accessible

### DISCUSSION THREAD

**Discussion by shaneahmed** [pradyunsg/furo#514] [Potential Issue] [answered]
The index page is not being generated on readthedocs.
If I use `alabaster` theme the index page is generated and is accessible. 
But with Furo theme it is not accessible.

Please see the links below:
https://tia-toolbox.readthedocs.io/en/doc-change-theme/#
https://tia-toolbox.readthedocs.io/en/develop/#

If I navigate to another page, clicking on top left takes me to index.html page on develop with alabaster theme. Whereas with furo theme index page is not accessible. 

Attached both links
https://tia-toolbox.readthedocs.io/en/develop/index.html
https://tia-toolbox.readthedocs.io/en/doc-change-theme/index.html

This is being tested in this PR https://github.com/TissueImageAnalytics/tiatoolbox/pull/457

**Comment by pradyunsg** [comment, marked as answer]
I'm not sure I follow. I certainly see the pages that you've linked to.

![Screenshot 2022-09-02 at 19 26 19](https://user-images.githubusercontent.com/3275593/188215445-56a289c9-8a91-41b4-83e2-c5bb0c94b952.png)
![Screenshot 2022-09-02 at 19 27 27](https://user-images.githubusercontent.com/3275593/188215624-4406afcf-dd0d-4e85-9ac4-3ad2d0e5bc10.png)

**Reply by shaneahmed** [reply]
Sorry for not being clear. I removed some of the flags and it is working now. 

The issue was that https://tia-toolbox.readthedocs.io/en/doc-change-theme/index.html was not being generated but now it is accessible.

Thanks for designing this theme. It is brilliant!
